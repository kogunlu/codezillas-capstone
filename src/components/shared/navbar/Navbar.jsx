import React from 'react';
import {Link } from 'react-router-dom';
import Button from '../button/Button';


function Navbar() {
  return (
    <div className='nav-bg'>
      <div className='p-3'>
        <div className='grid grid-cols-4'>
          <div className='grid grid-cols-3 col-span-2'>
            <Link to="/">
              <div className='grid grid-cols-2 ml-20'>
                <img src="/logo.png" alt='Healing logo' />
                <h1 className='sm:text-xl md:text-5xl' >Healing</h1>
              </div>
            </Link>
          </div>
          <ul className='grid grid-cols-6 col-span-2 content-center '>
            <li className='py-2'><Link className='hover:text-yellow-white' to="/">Home</Link></li>
            <li className='py-2'><Link className='hover:text-yellow-white' to="/blogs">Blogs</Link></li>
            <li className='py-2'><Link className='hover:text-yellow-white' to="/about">About Us</Link></li>
            <li className='py-2'><Link className='hover:text-yellow-white' to="/contact">Contact Us</Link></li>
            <li><Button className="col-span-2" type='button' name="Login" classes="py-2 px-5  hover:bg-cyan-200 bg-cyan-400 font-medium rounded text-sm md:text-lg"/></li>
          </ul>
        </div>
      </div>
    </div>
  )
}

export default Navbar