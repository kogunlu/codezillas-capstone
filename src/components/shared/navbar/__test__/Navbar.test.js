import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../Navbar';



test('Navbar component renders correctly', () => {
  const tree = renderer.create(<BrowserRouter><Navbar/></BrowserRouter>).toJSON();
  expect(tree).toMatchSnapshot();
});
