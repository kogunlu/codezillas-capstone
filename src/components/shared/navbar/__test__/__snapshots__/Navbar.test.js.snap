// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar component renders correctly 1`] = `
<div
  className="nav-bg"
>
  <div
    className="p-3"
  >
    <div
      className="grid grid-cols-4"
    >
      <div
        className="grid grid-cols-3 col-span-2"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <div
            className="grid grid-cols-2 ml-20"
          >
            <img
              alt="Healing logo"
              src="/logo.png"
            />
            <h1
              className="sm:text-xl md:text-5xl"
            >
              Healing
            </h1>
          </div>
        </a>
      </div>
      <ul
        className="grid grid-cols-6 col-span-2 content-center "
      >
        <li
          className="py-2"
        >
          <a
            className="hover:text-yellow-white"
            href="/"
            onClick={[Function]}
          >
            Home
          </a>
        </li>
        <li
          className="py-2"
        >
          <a
            className="hover:text-yellow-white"
            href="/blogs"
            onClick={[Function]}
          >
            Blogs
          </a>
        </li>
        <li
          className="py-2"
        >
          <a
            className="hover:text-yellow-white"
            href="/about"
            onClick={[Function]}
          >
            About Us
          </a>
        </li>
        <li
          className="py-2"
        >
          <a
            className="hover:text-yellow-white"
            href="/contact"
            onClick={[Function]}
          >
            Contact Us
          </a>
        </li>
        <li>
          <input
            className="py-2 px-5  hover:bg-cyan-200 bg-cyan-400 font-medium rounded text-sm md:text-lg"
            onClick={[Function]}
            type="button"
            value="Login"
          />
        </li>
      </ul>
    </div>
  </div>
</div>
`;
